docker-compose.yml

```diff
@@ -14,8 +14,8 @@ services:
       - .env
     environment:
       API_HOST: api
-      BDB_NODE_HOST: "${BDB_NODE_HOST}"
-      BDB_NODE_PORT: "${BDB_NODE_PORT}"
+      BDB_NODE_HOST: bigchaindb
+      BDB_NODE_PORT: 9984
```

Comment:
Missed this earlier when it was changed, but this shouldn't be breaking anything. `bigchaindb` is the service name of the container running the BigchainDB server so docker should know to hook up to it directly.